bigbluebutton-client/src/org/bigbluebutton/modules/whiteboard/views/WhiteboardCanvas.as

```diff
@@ -370,13 +375,19 @@ package org.bigbluebutton.modules.whiteboard.views {
 			if (e.whiteboardId == currentWhiteboardId) {
 				whiteboardToolbar.whiteboardAccessModified(e.multiUser);
 				canvasModel.multiUserChange(e.multiUser);
-				canvasDisplayModel.multiUserChange(e.multiUser);
+				
+				if (!e.multiUser) {
+					canvasDisplayModel.clearCursors();
+				}
 			}
 		}
 		
 		private function onReceivedCursorPosition(e:WhiteboardCursorEvent):void {
-			if (e.whiteboardId == currentWhiteboardId) {

```

Comment:
Removing this check means that you're going to be bleeding cursors between pods. Please recheck everything with pods.